src/_guides/language/analysis-options.md

```diff
@@ -293,12 +293,11 @@ linter:
     await_only_futures: true
 ```
 
-{{site.alert.note}}
-  **If you disable any rules, then use key-value syntax to enable rules.**
-  For example, the preceding example can’t use `- await_only_futures`;
-  instead, it must use `await_only_futures: true`.
-  The reason it can’t use the dash shortcut (`-`) to enable `await_only_futures`
-  is that YAML doesn't support mixing list and key-value syntaxes.
+{{site.alert.important}}
+  **Don't mix list and key-value syntaxes in the same options file.**
```

Comment:
in -> for linter rules in
?
I think you can mix the syntaxes, as long as they aren't modifying the same tag.

Reply:
You can't even if they're different tags as it would give a parsing error

Reply:
Oh, that's so counterintuitive! @pq can you confirm?

Reply:
Hmmm. IIRC @KNawm is right. 😬

Reply:
I'd be really surprised if such a constraint existed. In fact, the Dart analyzer (2.5.1) is perfectly happy with the following YAML, which contains both syntaxes:

```yaml
include: package:pedantic/analysis_options.yaml

linter:
  rules:
    camel_case_types: false

analyzer:
  exclude:
    - path/to/excluded/files/**
    - some_other_path/file.*
```

```terminal
$ dartanalyzer analysis_options.yaml .
Analyzing analysis_options.yaml, ...
No issues found!
```

Can you give an example of the kind of error that you are getting?

Reply:
Oh, I was talking about the rules collection specifically, so maybe it isn't clear enough.
I'm talking about this:
```yaml
linter:
  rules:
    - annotate_overrides
    - await_only_futures
    - camel_case_types
    avoid_shadowing_type_parameters: false
    await_only_futures: true
```